### PR TITLE
CASMTRIAGE-6678 1.5 : update doc references for winter release

### PR DESCRIPTION
**Pull request by kimjensen-hpe**
# Description
Resolves [CASMTRIAGE-6678](https://jira-pro.it.hpe.com:8443/browse/CASMTRIAGE-6678) : DOCS: Slurm and PBS service recovery should point to the correct documentation name

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/System_Recovery/PBS_Service_Recovery.md [hunk 1]
```diff
@@ -10,9 +10,8 @@ The following covers restoring PBS data.
 
 ## Service recovery for PBS
 
-To restore PBS Professional data from backup, follow section
-*10.7.7 Restore PBS home directory from backup* in the
-**HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX Systems (S-8003)**.
+To restore Portable Batch System data from backup, see *Restore PBS home directory from backup* in the
```
Comment (jld-dehn-hpe) [inline operations/System_Recovery/PBS_Service_Recovery.md]:
```suggestion
To restore Portable Batch System data from a backup, see *Restore PBS home directory from a backup* in the
```

File: operations/System_Recovery/Slurm_Service_Recovery.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ The following covers restoring Slurm data.
 
 ## Service recovery for Slurm
 
-To restore Slurm data from backup, follow sections
-*10.3.11 Restore Slurm Accounting Database from Backup* and
-*10.3.13 Restore Slurm Spool Directory from Backup* in the
-**HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX Systems (S-8003)**.
+To restore Slurm data from backup, see
+*Restore Slurm accounting database from backup* and
+*Restore Slurm spool directory from backup* in the
```
Comment (jld-dehn-hpe) [inline operations/System_Recovery/Slurm_Service_Recovery.md:15]:
```suggestion
*Restore Slurm accounting database from a backup* and
*Restore Slurm spool directory from a backup* in the
```

File: operations/System_Recovery/Slurm_Service_Recovery.md [hunk 3]
```diff
@@ -10,10 +10,10 @@ The following covers restoring Slurm data.
...
+**HPE Slurm Installation Guide: CSM on HPE Cray EX Systems (S-8058)**.
```
Comment (jld-dehn-hpe) [inline operations/System_Recovery/Slurm_Service_Recovery.md]:
```suggestion
To restore Slurm data from a backup, see
*Restore Slurm accounting database from a backup* and
*Restore Slurm spool directory from a backup* in the
**HPE Slurm Installation Guide: CSM on HPE Cray EX Systems (S-8058)**.
```

File: operations/System_Recovery/PBS_Service_Recovery.md (hunk 1 above)
Comment (jld-dehn-hpe) [reply operations/System_Recovery/PBS_Service_Recovery.md]: Added a few minor changes from the CSM PRs I'm working on.

File: operations/System_Recovery/Slurm_Service_Recovery.md (hunk 3 above)
Comment (jld-dehn-hpe) [reply operations/System_Recovery/Slurm_Service_Recovery.md]: Added a few minor changes from the CSM PRs I'm working on.

File: operations/System_Recovery/PBS_Service_Recovery.md (hunk 1 above)
Comment (kimjensen-hpe) [reply operations/System_Recovery/PBS_Service_Recovery.md]: I'm basing that string on the section name in the actual doc, so I think leaving it w/o the added " a " is best.

File: operations/System_Recovery/Slurm_Service_Recovery.md (hunk 3 above)
Comment (kimjensen-hpe) [reply operations/System_Recovery/Slurm_Service_Recovery.md]: I'm basing that string on the section name in the actual doc, so I think leaving it w/o the added " a " is best.

File: operations/System_Recovery/PBS_Service_Recovery.md (hunk 1 above)
Comment (jld-dehn-hpe) [reply operations/System_Recovery/PBS_Service_Recovery.md]: @kimjensen-hpe, here's the PR with the added "a" in the section titles of the document: https://github.hpe.com/hpe/hpc-wlm-pbs-docs/pull/31. I'll likely be merging this PR today. A PDF version is attached for reference.

[PBS_CSM_Install_Guide_revised.pdf](https://github.com/Cray-HPE/docs-csm/files/14388711/PBS_CSM_Install_Guide_revised.pdf)

I'll be making this same change in a Slurm PR later today.

**Review by jld-dehn-hpe — APPROVED** [review]

**Review by bo-quan — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**